resource_alks_iamrole.go

```diff
@@ -199,7 +199,7 @@ func resourceAlksIamRoleRead(ctx context.Context, d *schema.ResourceData, meta i
 		//If error is 404, RoleNotFound, we log it and let terraform decide how to handle it.
 		//All other errors cause a failure
 		if err.StatusCode == 404 {
-			log.Printf("[Error] %s", err.Err)
+			log.Printf("[Error] %s", err)
```

Comment:
I think in this case we will need to print err.Error(), since printf is expecting a string, not an error object.

Reply:
good catch